examples/bridge-app/esp32/main/main.cpp

```diff
@@ -328,6 +329,15 @@ void HandleDeviceStatusChanged(Device * dev, Device::Changed_t itemChangedMask)
     }
 }
 
+bool emberAfBridgedActionsClusterInstantActionCallback(app::CommandHandler * commandObj,
+                                                       const app::ConcreteCommandPath & commandPath,
+                                                       const BridgedActions::Commands::InstantAction::DecodableType & commandData)
+{
+    // No actions are implemented
```

Comment:
This is needed since the zap generated files are common, and instant action is enabled in the zap file, and adding individual actions can happen at run time. Since no actions are implemented I changed it to return Not Found.

Reply:
That's a start, but per spec the command should not be present at all if no actions are available, if I read the spec right.  If actions can be added at run time, that's a problem.  @leorozendaal, could you please take a look?